modules/subscriber/queue.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_sqs_queue" "dead_letter" {
+  name = "${var.name}-deadletter"
+}
+
+resource "aws_sqs_queue" "queue" {
+  name                      = var.name
+  delay_seconds             = 0
+  message_retention_seconds = 1209600
+  redrive_policy = jsonencode({
+    deadLetterTargetArn = aws_sqs_queue.dead_letter.arn
+    maxReceiveCount     = 4
+  })
+  visibility_timeout_seconds = 20
```

Comment:
This needs to be tied to the lambda execution time. Take a look at the forwarder module to see how it was configured.